### DISCUSSION TITLE
［求助］一觉醒来发现Linux（deepin）登录失败！

### DISCUSSION THREAD

**Discussion by Yizakl** [share121/inter-knot#2968] [其他]
![IMG_3326](https://github.com/user-attachments/assets/d09bb976-cdcb-4757-82c6-0d6bccec3315)
本人双系统（Windows11+Deepin V23）此前两个系统一切正常，早上一登录Linux就会弹这个命令行界面，然后再次跳转至输入密码界面
（密码是对的，在tty2中登录成功）

**Comment by Tartaglia770** [comment]
我擦，刚刚

**Comment by Legendary-agent** [comment]
这个委托我接不了。

**Comment by hktklml** [comment]
没想到，这里也有深友，厉害

**Comment by hktklml** [comment]
我想知道23是哪一个版本，是beta还是正式

**Reply by Yizakl** [reply]
已经历经 3 年的筹备，迭代了 9 个版本、经历了 51 次内测，15日就正式发布了（是正式版）

**Reply by hktklml** [reply]
尝试一下sudo apt install intel-microcode或sudo apt install amd64-microcode

**Comment by Buer-Nahida** [comment]
我是NixOS，对deepin不熟悉，翻找社区后发现一些人执行下面两行命令后就修复了，你可以试试
```bash
sudo apt install kde-plasma-desktop
sudo dpkg-reconfigure lightdm
```

**Comment by VacuolePaoo** [comment]
![image](https://github.com/user-attachments/assets/e34889e1-7ebc-498a-a21b-2e5479fe827e)
帮你问懂得人了
